cmd/metrics-v3-cache.go

```diff
@@ -18,12 +18,21 @@
 package cmd
 
 import (
+	"sync"
 	"time"
 
 	"github.com/minio/madmin-go/v3"
 	"github.com/minio/minio/internal/cachevalue"
 )
 
+var (
+	// prevDriveIOStats is used to calculate "per second"
+	// values for IOStat related disk metrics e.g. reads/sec.
+	prevDriveIOStats            map[string]madmin.DiskIOStats
+	prevDriveIOStatsMu          sync.RWMutex
+	prevDriveIOStatsRefreshedAt time.Time
+)
+
```

Comment:
I suggest putting these in line 130 (beginning of `newDriveMetricsCache`) as it is only meant for the `loadDriveMetrics` closure there - mainly to reduce bringing in new globals.

Reply:
thanks, done